### DISCUSSION TITLE
install packages

### DISCUSSION THREAD

**Discussion by joejnt** [teamalgoritma/community#805] [Q&A] [answered]
Dear rekan2 Algoritma,

Mohon bantuannya saya dapat permasalahan untuk install packages rmdformats. Proses instal nya berhasil tapi saat mau dipanggil library nya muncul seperti dibawah ini.
![library_rmdformats](https://github.com/teamalgoritma/community/assets/103427989/b861cf3e-d320-4ce8-a7e5-c6e44eeab79d)

Saya sudah coba dengan remove packages rmdformats dan remova packages xfun, kemudian install kembali packages xfun dan rmdformats. Namun tidak berhasil. 

Mohon sarannya.

Best regards,
JT

**Comment by IrfanChairurrachman** [comment, marked as answer]
Dear @joejnt ,

Dari error yang terlihat tertulis bahwa `rmdformats` membutuhkan library `xfun` dengan version di atas 0.39. Mungkin boleh Pak di screenshot setelah menginstall `xfun` versionnya berapa? bisa dicari di bagian package ya Pak

![image](https://github.com/teamalgoritma/community/assets/56690035/08c274e2-cd29-422c-bf2b-b634144cee9e)

Nah, jika telah di-uninstall lagi (dengan klik tombol silang di paling kanan) Bapak bisa menginstall lagi dengan version yang diinginkan, dalam contoh di atas yaitu 0.40. Untuk instalasi dengan version yang diinginkan bisa dengan cara berikut.

`install.packages("xfun", version='0.40')`

Setelah itu jangan lupa untuk restart R session dengan cara Session > Restart R, seperti berikut:
![image](https://github.com/teamalgoritma/community/assets/56690035/b6cbbf67-9e36-4525-8a33-888050a47746)

Setelah itu silahkan untuk load library `rmdformats` lagi.

Sekian jawaban dari kami. Bila ternyata informasi ini dapat menjawab pertanyaan Anda, Anda dapat “Mark as answer” jawaban ini.

**Comment by kinantytasyao** [comment]
Halo @joejnt  , karena tidak ada konfirmasi lebih lanjut dan untuk menjaga agar forum diskusi tetap berjalan kondusif, kami izin melakukan lock/close diskusi ini. Semoga dapat dipahami.
